### ISSUE TITLE
Signaling connection disconnected when trying to connect to room

### ISSUE THREAD

**Issue by immexerxez** [twilio/twilio-video.js#48]
I'm having an issue connecting to a room that I am giving a random string name. The client connects fine but I get error 53001 when trying to connect to a room which I also saw wasn't in the twilio error code manual. Any suggestions? I tried the started project and it worked perfectly.

`var request = $.ajax({
      url: "/token",
      type: "POST",
      dataType: "json",
      data: {symptoms : arrayIt},
      
      success: function(response) {
        identity = response.identity;
        toke = response.token;
        roomName = response.identifier;

        videoClient = new Twilio.Video.Client(toke);
        if (videoClient)
            videoClient.connect({ to: roomName}).then(roomJoined,
            function(error) {
              log('Could not connect to Twilio: ' + error.message);
            });
      },
      error: function( errorThrown){
        console.log(errorThrown);
        }
    });`

**Comment by markandrus** [comment]
Hi  @immexerxez,

Sorry to hear that! When you construct your `Twilio.Video.Client`, can you construct it passing the `logLevel` parameter? Like so:

```js
videoClient = new Twilio.Video.Client(toke, { logLevel: 'debug' });
```

and then paste the resulting logs? This should allow me to see the Access Token you pass in and anything weird that might be happening.

Thanks,
Mark

**Comment by immexerxez** [comment]
Hi @markandrus thank you for your response. I did as instructed and I think The issue is here 
`Sun Feb 05 2017 22:30:16 GMT-0600 (CST) | sip.transport | received WebSocket text message:

SIP/2.0 400 Bad request
CSeq: 8042 INVITE
Call-ID: 378vv1117scc5cbi8ogh
From: <sip:undefined@AC73c5c5f676305d93b49daaa7203405c7.endpoint.twilio.com>;tag=s3mgohirmt
To: <sip:orchestrator@AC73c5c5f676305d93b49daaa7203405c7.endpoint.twilio.com>;tag=37783021_6772d868_9e414ad9-89e2-44b3-b6c6-0557e5e4fa75
Via: SIP/2.0/WSS 127.0.0.1:51706;branch=z9hG4bK919890;rport=51706
Server: Twilio
Contact: <sip:172.18.2.5:443;transport=wss>
Supported: room-signaling
Require: timer
Session-Expires: 120;refresher=uac
Content-Type: application/room-signaling+json
X-Twilio-CallSid: CA07d06f5f64e16023200a874ae3ca144f
Content-Length: 113

{"version":1,"type":"error","participants":[],"error":{"code":53200,"message":"Participant identity is invalid"}}`

if this is not sufficient I will gladly add the rest of the log. but it seems that I am setting the identity incorrectly or with a null value.

**Comment by immexerxez** [comment]
@markandrus thank you for notifying me about adding "loglevel debug" I discovered that I was incorrectly passing the identity of the user requesting a token.

**Comment by markandrus** [comment]
@immexerxez I am glad it worked out!

**Comment by mjamal-tiu** [comment]
I am also getting same email see more details at below ticket and provide me any solution.
https://github.com/twilio/twilio-video.js/issues/576

Thanks in advanced

**Comment by mjamal-tiu** [comment]
Issue is resolved based on suggestion provided on #576
